ratis-server/src/main/java/org/apache/ratis/server/DisabledDataStreamServerFactory.java

```diff
@@ -33,6 +34,11 @@ public DataStreamServerRpc newDataStreamServerRpc(RaftServer server) {
       @Override
       public void start() {}
 
+      @Override
+      public InetSocketAddress getInetSocketAddress() {
+        return InetSocketAddress.createUnresolved("0.0.0.0", 0);
```

Comment:
Is it okay to return null?

Reply:
hmm good point. Let me try.

Reply:
`null` works. Have updated to using `null`